maze-game/SpeedPotion_option_1.java

```diff
@@ -11,7 +11,7 @@ public class SpeedPotion_option_1 extends Actor
     public SpeedPotion_option_1()
     {
         GreenfootImage inv_1 = getImage();
-        inv_1.scale(95, 95);
```

Comment:
I made it 95 95 scale here because i thought it fit a little nicer in the inventory that way, but dw